web/sites/default/config/core.entity_form_display.media.document.default.yml

```diff
@@ -37,6 +57,14 @@ content:
     third_party_settings: {  }
     type: chosen_select
     region: content
+  field_efiles_link:
+    weight: 8
+    settings:
+      placeholder_url: ''
+      placeholder_title: ''
```

Comment:
@mike-mcdonald, I'm not sure we want to give them the option to override the title. It is possible to turn of the link label per link field. I think that would be appropriate for this type of link. Additionally, we may want to restrict the field to only allow links from the efiles domain—which I believe is also a configuration option.